### ISSUE TITLE
Support for cucumber command line arguments (e.g., tags)

### ISSUE THREAD

**Issue by john-fitzpatrick** [ngauthier/hydra#9]
There are some useful cucumber command line arguments, like tags and "--guess" that would be useful to be able to pass through to the runner in some fashion.

**Comment by ngauthier** [comment]
I agree completely. This is on my list, but I may not get to it for a while as we don't need this feature day-to-day.

If you're interested in implementing it, the steps to take would be:

1) Add a hydra/tasks.rb configuration for cucumber opts
2) Pass that down from the master to the worker and then the children in some sort of boot-config message
3) In the runner's "run_cucumber_file" method, add in the opts to the existing options.

**Comment by bowsersenior** [comment]
This would be a great feature to have. I'd love to help out implementing it, but I'm new to hydra and this seems tricky.
